src/java/Light_vehicle.java

```diff
@@ -0,0 +1,36 @@
+package java;
+
+import java.util.Scanner;
+
+public class Light_vehicle extends Land_transport{
+    String TypeOfBody;
+    int NumberOfSeats;
+    private int travel;
+    private double a;
+    private int time;
+
+
+    @Override
+    void Vyvod() {
+        super.Vyvod();
+        System.out.println("Type of body: " + TypeOfBody);
+        System.out.println("Number of seats: " + NumberOfSeats);
+    }
+    private void rashod(){
+         a =travel/FuelConsumption;
```

Comment:
@Hlieb Операторы и пробелы
```suggestion
         a = trave l/ FuelConsumption;
```